cognite/client/_api/entity_matching.py

```diff
@@ -0,0 +1,221 @@
+
+        Args:
+            sources: entities to match from, should have an 'id' field. Tolerant to passing more than is needed or used (e.g. json dump of time series list)
+            targets: entities to match to, should have an 'id' field.  Tolerant to passing more than is needed or used.
+            id_field (str): use 'id' or 'external_id' as the id field to match resources
+            true_matches: Known valid matches given as a list of dicts with keys 'sourceId', 'sourceExternalId', 'sourceId', 'sourceExternalId'). If omitted, uses an unsupervised model.
+             A tuple can be used instead of the dictionary for convenience, interpreted as id/externalId based on type.
+            match_fields: List of (from,to) keys to use in matching. Default in the API is [('name','name')]
+            feature_type (str): feature type that defines the combination of features used, see API docs for details.
+            classifier (str): classifier used in training. Currently undocumented in API.
+            ignore_missing_fields (bool): whether missing data in keyFrom or keyTo should return error or be filled in with an empty string. Currently undocumented in API.
+            name (str): Optional user-defined name of model.
+            description (str): Optional user-defined description of model.
+            external_id (str): Optional external id. Must be unique within the project.
+        Returns:
+            EntityMatchingModel: Resulting queued model."""
+
+        if match_fields:
+            match_fields = [ft if isinstance(ft, dict) else {"source": ft[0], "target": ft[1]} for ft in match_fields]
+        if true_matches:
+            true_matches = [convert_true_match(true_match) for true_match in true_matches]
+        response = self._post(
+            self._RESOURCE_PATH + "/",
+            json={
+                "name": name,
+                "description": description,
+                "externalId": external_id,
+                "sources": EntityMatchingModel.dump_entities(sources),
+                "targets": EntityMatchingModel.dump_entities(targets),
+                "trueMatches": true_matches,
+                "matchFields": match_fields,
+                "featureType": feature_type,
+                "classifier": classifier,
+                "ignoreMissingFields": ignore_missing_fields,
+            },
+        )
+        return self._LIST_CLASS._RESOURCE._load(response.json(), cognite_client=self._cognite_client)
+
+    def predict(
+        self,
+        sources: Optional[List[Dict]] = None,
+        targets: Optional[List[Dict]] = None,
+        num_matches=1,
+        score_threshold=None,
+        id: Optional[int] = None,
+        external_id: Optional[str] = None,
+    ) -> ContextualizationJob:
+        """Predict entity matching. NB. blocks and waits for the model to be ready if it has been recently created.
```

Comment:
NB? Note well? Maybe expand this a bit

Reply:
https://dictionary.cambridge.org/dictionary/english/nb